SECURITY.md

```diff
@@ -0,0 +1,47 @@
+# Security Policies and Procedures
+
+This document outlines security procedures and general policies for the SORMAS
+project.
+
+  * [Reporting a Security Bug](#reporting-a-security-bug)
+  * [Disclosure Policy](#disclosure-policy)
+  * [Comments on this Policy](#comments-on-this-policy)
+  
+If you want to report a bug which is not security sensible, please [submit an issue](https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/CONTRIBUTING.md#submitting-an-issue). 
+
+## Reporting a Security Bug
+
+Our team and community take all security bugs in SORMAS seriously.
+Thank you for improving the security of SORMAS. We appreciate your efforts and
+responsible disclosure and will make every effort to acknowledge your
+contributions.  
+Unfortunately, SORMAS does not offer a paid bug bounty programme or other forms of compensation.
+
+Report security bugs by emailing at **security@sormas.org**.
+
+We will acknowledge your email within 48 hours, and will send a
```

Comment:
We will acknowledge your email and follow up with a response within 10 business days, or explain why a reply may take longer. The response will indicate the next steps in handling your report.